app/furniture/section_navigation/section_navigations/_section_navigation.html.erb

```diff
@@ -1,10 +1,7 @@
 <div class="mt-3 grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4 items-center">
   <% policy_scope(section_navigation.rooms).each do |room| %>
-    <%= link_to [room.space, room], class: "no-underline" do %>
-      <%= render CardComponent.new(
-        data: { access_level: room.access_level, slug: room.slug, model: "room", id: room.id },
-        classes: "group self-stretch hover:bg-orange-50"
-        ) do %>
+    <%= link_to polymorphic_path(room.location), class: "no-underline" do %>
```

Comment:
I'm not sure why, but `link_to` is including the full url; which I think is related to our over-loading of `url_for`. 

I didn't want to get into it; but it's annoying.